apps/web/lang/de.json

```diff
@@ -171,7 +171,7 @@
   "quantitySelectorIncrease": "Menge erhöhen",
   "scrollTop": "Nach oben scrollen",
   "numberInCart": "{count} Produkte im Warenkorb",
-  "languageSelector": "Sprachauswahl",
+  "languageSelector": "Klicken Sie hier, um den modalen LanguageSelector zu öffnen",
```

Comment:
Why are we using a new translation here?

Reply:
if you search "languageSelector" in project this word is not use, so i changed it to match provided mockups